src/Sku.php

```diff
@@ -5,21 +5,53 @@
 use Macaroons\Macaroon;
 use Macaroons\Packet;
 
-class Sku {
-    public static function generateSKUToken($secret, $sku, $amount, $currency, $description, $expiry) {
-       $identifier = "Brave SKU v1.0";
-       // Current website or variable representing the website
-       $location = $_SERVER['HTTP_HOST'];
+class Sku
+{
+    private static function getHost()
+    {
+        return isset($_SERVER['HTTP_HOST']) ? $_SERVER['HTTP_HOST'] : '';
+    }
+
+    public static function generateSKUToken($secret, $sku, $amount, $currency, $description, $expiry)
+    {
+        $identifier = "Brave SKU v1.0";
+        // Current website or variable representing the website
+        $location = Sku::getHost();
+
+        $m = new Macaroon($secret, $identifier, $location);
+    
+        $m->addFirstPartyCaveat('sku = ' . $sku);
+        $m->addFirstPartyCaveat('amount = ' . $amount);
+        $m->addFirstPartyCaveat('currency = ' . $currency);
+        $m->addFirstPartyCaveat('description = ' . $description);
+        $m->addFirstPartyCaveat('expiry = ' . $expiry);
 
-       $m = new Macaroon($secret, $identifier, $location);
+        return $m;
+    }
+
+    private static function sendResponse($status, $message)
+    {
+        $response = [
+            'status' => $status,
+            'message' => $message
+        ];
+        return $response;
+    }
 
-       $m->addFirstPartyCaveat('sku = ' . $sku);
-       $m->addFirstPartyCaveat('amount = ' . $amount);
-       $m->addFirstPartyCaveat('currency = ' . $currency);
-       $m->addFirstPartyCaveat('description = ' . $description);
-       $m->addFirstPartyCaveat('expiry = ' . $expiry);
+    public static function validateOrderStatus($orderId)
+    {
+        $location = Sku::getHost();
+        $url = "https://grant.rewards.brave.software/v1/orders/" . $orderId;
```

Comment:
Can we set this as a constant somewhere? I think eventually we would allow partners to test in our staging environment or production environment?

Reply:
In environment variables? Update the env if you want to use production vs staging?